editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -1746,6 +1753,7 @@ AnimationPlayerEditor::AnimationPlayerEditor(EditorNode *p_editor, AnimationPlay
 	track_editor->set_v_size_flags(SIZE_EXPAND_FILL);
 	track_editor->connect("timeline_changed", callable_mp(this, &AnimationPlayerEditor::_animation_key_editor_seek));
 	track_editor->connect("animation_len_changed", callable_mp(this, &AnimationPlayerEditor::_animation_key_editor_anim_len_changed));
+	track_editor->connect("animation_step_changed", callable_mp(this, &AnimationPlayerEditor::_update_frame_step));
```

Comment:
I'd name it `_animation_key_editor_anim_step_changed` to match the callback for `animation_len_changed` (and place the method definition in header and .cpp next to that one).

Reply:
What about in places where it's not used as a callback (lines 299 & 1605)? Should it be renamed everywhere, or should `_animation_key_editor_anim_step_changed` be a new function that just calls `_update_frame_step`?

Reply:
Went with the latter since it seems less confusing to me, also just rebased to upstream.